lib/remap-lang/src/lib.rs

```diff
@@ -134,7 +134,7 @@ mod tests {
     #[test]
     fn it_works() {
         let cases: Vec<(&str, Result<Option<Value>>)> = vec![
-            (r#".foo = null || "bar""#, Ok(Some("bar".into()))),
+            (r#"$foo = null || "bar""#, Ok(Some("bar".into()))),
```

Comment:
Can we keep the path tests and add the variable tests to ensure it works for both cases?

Reply:
I'll add it, but note that these tests aren't that relevant right now. They mostly served me during development, the actual tests will be moved over from the Vector crate in the next couple of days.